### PR TITLE
MHV-59642 added sent folder to tab navigation, updated More folders t…

### PR DESCRIPTION
**Pull request by KyleCardwell**
…itle

**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

- Hot fix - Veteran feedback requested an easier way to navigate to sent messages, so we are adding a tab to the inner navigation of SM for "Sent" messages.  

### [MHV-59642](https://jira.devops.va.gov/browse/MHV-59642) - Hot fix: Implement design changes for viewing sent messages ###

> Hot fix: Implement design changes for viewing sent messages
> 
> https://www.figma.com/design/5uTyxzQa92awJV0dcVEhM6/X-linked-Internal-Navigation?node-id=2211-49353&t=EfTag74CMGi285dJ-0
> 
> Requirements are as follows: * Change "Folders" => "More folders"
> 
> - Don't need to change URL for now.
> - Update "folders" page title to "More folders"
> - Update breadcrumbs referencing "Folders" to read "More folders"
> - Update title tag of "More folders" page to "More folders - Messages | Veterans Affairs"
> - Changes to hanging links:
> - Add new "Sent" cross-link next to "Inbox"
> - Update link text for "Folders" link to read "More folders"
> - Remove "sent" from the list under "More folders"

## Testing done

- manual testing and updated unit tests

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |Screenshot 2024-07-03 at 8 44 32 AM|Screenshot 2024-07-03 at 8 44 24 AM|
| Desktop |Screenshot 2024-07-03 at 8 43 35 AM|Screenshot 2024-07-03 at 8 43 44 AM|

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/mhv-secure-messaging/tests/e2e/pages/FolderLoadPage.js [hunk 1]
```diff
@@ -64,6 +64,7 @@ class FolderLoadPage {
     this.loadFolderMessages('Drafts', -2, 1, messagesList);
   };
 
+  // this method no longer needed as sent folder link was moved to nav-bar
   loadSentMessages = (messagesList = mockSentMessages) => {
     this.loadFolderMessages('Sent', -1, 2, messagesList);
   };
```
Comment (vsaleem) [inline src/applications/mhv-secure-messaging/tests/e2e/pages/FolderLoadPage.js:70]: If this is no longer needed, maybe we can 86 this? cc: @fazilqa

**Review by vsaleem — APPROVED** [review]
Nice -- I had a comment for @fazilqa but other than that this pr looks great!

File: src/applications/mhv-secure-messaging/tests/e2e/pages/FolderLoadPage.js (hunk 1 above)
Comment (fazilqa) [reply src/applications/mhv-secure-messaging/tests/e2e/pages/FolderLoadPage.js:70]: I prefer to keep it for a bit more time and will remove it in my next cleanup PR

Comment (oleksii-morgun) [reply src/applications/mhv-secure-messaging/tests/e2e/pages/FolderLoadPage.js:70]: I see this is already merged. @fazilqa I believe we may want to discard any code in the future that is not planned to be used. we can always recover it from git version history 👍🏻

**Pull request merged**